repair_security/hook.py

```diff
@@ -0,0 +1,12 @@
+from odoo import SUPERUSER_ID
+from odoo.api import Environment
+
+
+def post_init_hook(cr, _):
+    env = Environment(cr, SUPERUSER_ID, {})
+    # Giving Repair Admin access to match the previous permissions.
+    env.ref("repair_security.group_repair_manager").write(
```

Comment:
Manager? shouldn't it be user?

Reply:
To match the previous permissions, you have to grant manager access to the Repair App. Basically I've added a Repair User without all the repair permissions.

Reply:
Fair enough!